cv_next/middleware.ts

```diff
@@ -1,13 +1,16 @@
 import { createServerClient, type CookieOptions } from "@supabase/ssr";
 import { NextResponse } from "next/server";
 import type { NextRequest } from "next/server";
-import { jwtVerify } from "jose";
 
 export async function middleware(request: NextRequest) {
   let response = NextResponse.next({
     request,
   });
 
+  if (request.nextUrl.pathname.startsWith("/actions/cvs/fetch")) {
```

Comment:
Here again, don't forget to remove hardcoded value

Reply:
Resolving this as I updated according to the last comment but want to hear your opinion on this - combining this and upper comment